content/en/05-reference/01-admin/09-uninstall.md

```diff
@@ -8,10 +8,37 @@ redirect_from:
     - /installing-pixie/uninstall/
 ---
 
-``` bash
+Pixie can be uninstalled with Pixie's [CLI tool](/installing-pixie/install-schemes/cli) or with `kubectl`.
 
-# Delete Pixie on the current K8s cluster:
+## Uninstall using the CLI (easiest)
 
+The easiest way to uninstall Pixie is with the CLI. To delete Pixie from the current cluster, run:
+
+```bash
 px delete
+```
+
+Note that `px delete` will not delete the `olm` namespace. During installation, Pixie will install the [Operator Lifecycle Manager](https://docs.openshift.com/container-platform/4.5/operators/understanding/olm/olm-understanding-olm.html) (OLM) if it is not already present. Make sure that no other applications are using the OLM before manually deleting it with `kubectl delete namespace olm`.
+
+## Uninstall using `kubectl`
+
+To delete Pixie using `kubectl`, run the following commands.
 
+These commands assume that Pixie's components were deployed to their default namespaces.
+
+```bash
+# Delete Pixie operator.
+kubectl delete namespace px-operator
+
+# Delete Pixie vizier.
+kubectl delete namespace pl
+
+# Delete Pixie cloud.
```

Comment:
I would leave a note that this step is not necessary if they just want to remove the Vizier running in the cluster.